scout/adapter/mongo/panel.py

```diff
@@ -546,28 +546,30 @@ def clinical_hgnc_ids(self, case_obj):
         )
         return set(item["_id"] for item in query_result)
 
-    def search_panels_hgnc_id(self, hgnc_id):
-        """Return all panels and versions that contain given gene, list is sorted
-        Args:
-            self: PanelHandler()
-            hgnc_id: hgnsc_id (int) to search
-
-        Returns:
-             list(dict): example: [{_id: {display_name: 'panel1'}, versions: [1.0, 2.0]}, ...]
-        """
+    def search_panels_hgnc_id(self, hgnc_id: int) -> list[dict]:
+        """Return all panels and versions that contain given gene,list is sorted"""
         query = [
             {"$match": {"genes.hgnc_id": hgnc_id}},
             {
                 "$group": {
-                    "_id": "$display_name",
+                    "_id": {"name": "$display_name", "institute": "$institute"},
```

Comment:
Changed the code not to limit the search by institute, but to return institute info in the search result